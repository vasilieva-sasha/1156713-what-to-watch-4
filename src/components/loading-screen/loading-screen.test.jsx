import React from "react";
import renderer from "react-test-renderer";
import LoadingScreen from "./loading-screen";


it(`LoadingScreen`, () => {
  const tree = renderer.create(<LoadingScreen />).toJSON();

  expect(tree).toMatchSnapshot();
});
