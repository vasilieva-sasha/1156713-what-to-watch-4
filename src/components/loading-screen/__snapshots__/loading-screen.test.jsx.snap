// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="main.html"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    />
  </header>
  <div
    style={
      Object {
        "margin": "auto",
      }
    }
  >
    <p>
      Loading....
    </p>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="main.html"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
